src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -102,6 +102,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <PackageRequireLicenseAcceptance Condition="'$(PackageRequireLicenseAcceptance)'==''">false</PackageRequireLicenseAcceptance>
     <DebugSymbols Condition="'$(DebugSymbols)'==''">false</DebugSymbols>
     <CheckForOverflowUnderflow Condition="'$(CheckForOverflowUnderflow)'==''">false</CheckForOverflowUnderflow>
+
+    <WarningsAsErrors>NU1605</WarningsAsErrors>
```

Comment:
The reason we decided to do that was because the same properties are used by analyzers as well and for C#\VB those fields can have arbitrary error codes. From a user standpoint - it's easier to think of it as one place to suppress warnings coming from build than have one for each tool (given there's UI for these properties). 

We hadn't considered F#. Should we maybe default these only for C#\VB?